homeassistant/components/aurora/config_flow.py

```diff
@@ -9,7 +10,12 @@
 from homeassistant.core import callback
 from homeassistant.helpers import aiohttp_client
 
-from .const import CONF_THRESHOLD, DEFAULT_NAME, DEFAULT_THRESHOLD, DOMAIN
+from .const import (  # pylint: disable=unused-import
```

Comment:
Why is this needed? All these imports seem to be used.

Reply:
Had to add this due to a bug in PyLint which was causing DOMAIN to report as unused.